### DISCUSSION TITLE
GS-NSCBC boundary condition

### DISCUSSION THREAD

**Discussion by Pockjob** [AMReX-Combustion/PeleC#788] [Q&A]
Is the GS-NSCBC boundary condition available in current PeleC versions?

**Comment by marchdf** [comment]
It is not. However, it should not be too hard to revive it and we can definitely help you do that if you want to work on it and create a draft PR. Other solutions that are availble are: sponge layers and Blazek-type extrapolation.

**Comment by jrood-nrel** [comment]
There are branches where the NSCBC work was ported from our original Fortran implementation, but none of them were complete.
